lib/Indexer/Model/Index/IndexInfo.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Phpactor\Indexer\Model\Index;
+
+use Symfony\Component\Finder\SplFileInfo;
+
+class IndexInfo
+{
+    private string $absolutePath;
+
+    private string $directoryName;
+
+    private int $size;
+
+    private string $lastUpdated;
+
+    public function __construct(
+        string $absolutePath,
+        string $directoryName,
+        int $size,
+        string $lastUpdated
+    ) {
+        $this->directoryName = $directoryName;
+        $this->size = $size;
+        $this->lastUpdated = $lastUpdated;
+        $this->absolutePath = $absolutePath;
+    }
+
+    public static function create(SplFileInfo $fileInfo): self
+    {
+        return new self(
+            $fileInfo->getRealPath(),
+            $fileInfo->getRelativePathname(),
+            $fileInfo->getSize(),
+            sprintf('%.1f days', ((time() - $fileInfo->getMTime())/(3600*24)))
```

Comment:
lets keep the formatting lazy, better to have the raw data as class properties (i.e. `getMtime()` only)